app/src/main/java/org/p2p/wallet/striga/signup/ui/StrigaSignUpSecondStepPresenter.kt

```diff
@@ -88,7 +95,24 @@ class StrigaSignUpSecondStepPresenter(
         }
     }
 
+    private fun mapDataForStorage() {
+        cachedSignupData[StrigaSignupDataType.OCCUPATION]?.value?.let {
+            setCachedData(
+                type = StrigaSignupDataType.OCCUPATION,
+                value = it.uppercase()
+            )
+        }
+        cachedSignupData[StrigaSignupDataType.SOURCE_OF_FUNDS]?.value?.let {
+            // convert UI string into STRIGA_FORMAT
+            setCachedData(
+                type = StrigaSignupDataType.OCCUPATION,
+                value = it.split(" ").joinToString(separator = "_", transform = String::uppercase)
```

Comment:
`StrigaItemCellMapper` has the `toUiTitle`, you can add the same but with the opposite meaning: `fromUiTitle`